resources/js/app.js

```diff
@@ -105,6 +105,10 @@
                     return mapping[project.status];
                 }
 
+                $scope.sortProjects = function(){
```

Comment:
`(){` -> `() {` ?

It seems we have a mix of styles for this, so not necessary to fix it, or change it.

Need a linting task, including finding a suitable linter to lock down the style choice.

Reply:
> Need a linting task, including finding a suitable linter to lock down the style choice.

[JS Standard](https://github.com/standard/standard) linter used by many of the popular apps. It follows a standard sty;e guide and even it suggests a fix for this issue (`(){ -> () {`)